### DISCUSSION TITLE
Possible bug: underline not working in v3.0

### DISCUSSION THREAD

**Discussion by niloc7** [tailwindlabs/tailwindcss#6341] [Help] [answered]
I tried using ```underline``` everywhere, including the body, but nothing is being underlined.

But when I used this instead, it works.. 
```style="text-decoration: underline;"```

So it's not a problem with the browser or something.. 

Could this be a bug or are there other possible causes?

Note: All other class styles work just fine.

**Comment by niloc7** [comment, marked as answer]
Ahhh I just needed to restart the npm run watch as it ran into some errors. 

Not sure how to delete this topic.. but yeah, problem solved!

**Comment by merkdev** [comment]
It works very well actually
https://jsfiddle.net/gqxc16yk/
